ajax/libs/qcobjects/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "qcobjects",
+  "version": "2.1.298",
+  "description": "QCObjects a Quickcorp Objects Library - Cross Browser Javascript Framework for MVC Patterns",
+  "main": "qcobjects.min.js",
+  "license": "LGPL-3.0",
+  "npmName": "qcobjects",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "./QCObjects.js"
```

```suggestion
        "QCObjects.js"
```
Glob doesn't require './'

---

Done